blogs.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Page Title</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="mystyle.css">
+</head>
+<body>
+
+<div class="header">
+  <h1>My Website</h1>
+  <img src="./images/adoptadog-logo.svg" alt="Logo" width="1500" height="800">
+  <a href="./forms.html" class="button">Checkout</a>
+  <p>A website created by me.</p>
+</div>
+
+<div class="navbar">
+  <a href="index.html">Home</a>
+  <a href="./dogs.html">Dogs</a>
+  <a href="blogs.html">Blog</a>
+</div>
+<div class="row">
+  <div class="column">
+  <img src="./images/blog-1.jpg" alt="Girl With a Dog">
+  </div>
+
+  <div class="column">
+      <h3>Traveling With your Dog</h3>
+      <p>Some text</p>
+  </div>
+  </div>
+
+  <div class="row">
```

Comment:
how you're handling these blog posts might be a little problematic.

Instead, don't worry about rows and columns.  Think of how you could wrap all of the blog posts in a container div, and then inside each blog post lay out the information inside. This might be easier to explain in person though, definitely come see me and I can help you with the layout challenges.